pkg/apis/virt/v1alpha1/types.go

```diff
@@ -115,6 +116,11 @@ type InterfaceMasquerade struct {
 type InterfaceSRIOV struct {
 }
 
+type InterfaceVDPA struct {
+	NumQueues int  `json:"numQueues,omitempty"`
+	Iommu     bool `json:"iommu,omitempty"`
```

Comment:
Better name it IOMMU to keep it in Golang style

Reply:
The VdpaConfig struct and other structs in pkg/cloudhypervisor/client.go used Iommu too.
```golang
type VdpaConfig struct {
	Id         string `json:"id,omitempty"`
	Iommu      bool   `json:"iommu,omitempty"`
	NumQueues  int    `json:"num_queues"`
	Path       string `json:"path"`
	PciSegment int16  `json:"pci_segment,omitempty"`
}
```

Reply:
Yes, that's generated. For non-generated code, better keep them in style

Reply:
@fengye87 I updated the doc and IOMMU.